icu4c/source/i18n/olsontz.cpp

```diff
@@ -25,7 +25,7 @@
 #include "uassert.h"
 #include "uvector.h"
 #include <float.h> // DBL_MAX
-#include "uresimp.h" // struct UResourceBundle
+#include "uresimp.h" // struct UResourceBundle and StackUResourceBundle
```

Comment:
I think it'd be an even better idea to remove these comments altogether, they're too inconsistently used and too often not kept up-to-date to be truly useful.

Reply:
Thanks Fredrik, I'll push an update to just remove this comment entirely.